scripts/ci/stats/merged_prs.py

```diff
@@ -84,11 +74,28 @@ def main():
         prj['commits'] = pr.commits
         # The branch we are targeting. main vs release branches.
         prj['base'] = pr.base.ref
+        # defaults
+        prj['review_rule'] = "no"
+        prj['time_rule'] = False
+
+        labels = []
+        for label in pr.labels:
+            labels.append(label.name)
+        prj['labels'] = labels
+
+        reviewers = set()
+        for review in reviews:
+            # get list of all approved reviews
+            if review.user and review.state == 'APPROVED':
+                reviewers.add(review.user.login)
 
-        ll = []
-        for l in pr.labels:
-            ll.append(l.name)
-        prj['labels'] = ll
+        for assignee in pr.assignees:
+            if assignee.login in reviewers:
+                assignee_reviews += 1
+                # was reviewed at least by one assignee
+                prj['reviewed_by_assignee'] = "yes"
```

Comment:
it's never set to "no" anywhere? maybe also initialize with a default value alongside others? I guess default is "na"?

Reply:
Also just thinking out loud know, but I realize that Kibana (or any other reporting tool we'd want to hook up to the raw data, really) is probably smart enough to allow us to write the rules there, so wouldn't it be better to just make sure we capture *raw* data in ES (ex. "# of assignees", "# of approving assignees", etc), and add computed columns/fields in Kibana?

I guess having the rules visible here in the script is nice as it's easier for everyone to see and now how the dashbboards are then put together, but on the other it feels a bit like we might be making things more complex to update than necessary.

Reply:
> it's never set to "no" anywhere? maybe also initialize with a default value alongside others? I guess default is "na"?

removed completely, we have this already covered by `assignee_reviews` count.